velox/dwio/dwrf/common/FileMetadata.cpp

```diff
@@ -102,9 +102,9 @@ TypeKind TypeWrapper::kind() const {
     }
     case proto::orc::Type_Kind_CHAR:
     case proto::orc::Type_Kind_TIMESTAMP_INSTANT:
-      DWIO_RAISE(
+      VELOX_FAIL(fmt::format(
```

Comment:
this line break error msg.
```
Error Source: RUNTIME
Error Code: UNKNOWN
Reason: {} not supported yet.CHAR
Retriable: False
Context: Split [Hive: path_customer/000003_0 64101152 - 7199294] Task Gluten stage-3 task-2073
Top-Level Context: Same as context.
Function: kind
File: ../../velox/dwio/dwrf/common/FileMetadata.cpp
Line: 100
```

Reply:
cc @kerwin-zk

Reply:
@Yohahaha This [PR](https://github.com/facebookincubator/velox/pull/7310) maps the CHAR  type to VARCHAR, which is still under discussion.

Reply:
sorry, my mistake. this patch has no problem.